rollup.config.js

```diff
@@ -0,0 +1,13 @@
+const environment = require('./bundlers/environment')
+const developmentConfig = require('./bundlers/rollup.development')
+const productionConfig = require('./bundlers/rollup.production')
+
+const configs = []
```

Comment:
`const config = environment.isDev ? developmentConfig : productionConfig;
`?

Reply:
ternary is bad

Reply:
and configs will be used for any additional configs of environment, for example testConfig